### PR TITLE
Update Lightning-Tensor install docs

### PR DESCRIPTION
**Pull request by DSGuala**
### Before submitting

**Context:**
Lightning tensor docs did not describe installation via `pip install`

**Description of the Change:**

- Added instruction to `pip install cutensornet-cu12` and `pip install pennylane-lightning-tensor`
- Added warning about installing from source.

**Benefits:**
Clearer installation docs.

**Possible Drawbacks:**
Duplication of information in docs and pennylane.ai/install

**Related GitHub Issues:**

### CODE REVIEW

**Comment by multiphaseCFD** [comment]
Thanks @DSGuala ! LGTM. Could you please also help to update this line [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/035b63de739fd1973244c9942b721711b6ecdd5a/doc/lightning_tensor/device.rst?plain=1#L56) by change the single quotation of the `lightning.tensor` to a double - quotation one? Thank

File: README.rst [hunk 1]
```diff
@@ -383,8 +383,20 @@ Lightning-Tensor requires CUDA 12 and the `cuQuantum SDK <https://developer.nvid
 The SDK may be installed within the Python environment ``site-packages`` directory using ``pip`` or ``conda`` or the SDK library path appended to the ``LD_LIBRARY_PATH`` environment variable.
 Please see the `cuQuantum SDK <https://developer.nvidia.com/cuquantum-sdk>`_ install guide for more information.
 
+Lightning-Tensor and cutensornet can be installed via:
```
Comment (maliasadi) [inline README.rst:386]:
```suggestion
Lightning-Tensor and ``cutensornet-cu12`` can be installed via:
```

**Review by maliasadi — APPROVED** [review]
Thanks @DSGuala!

**Review by AntonNI8 — APPROVED** [review]

**Review by multiphaseCFD — APPROVED** [review]
Thanks @DSGuala !

**Pull request merged**